### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Feature_Create New File_Screen Reader: Narrator is not announcing the role as "Combo box" for the control "Kind"

### ISSUE THREAD

**Issue by Manisha370** [microsoft/azure-tools-for-java#3571]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-NewFeature; #MAS1.3.1; #ScreenReader; #C+E-H; #JavatoolingforIntelliJ/Eclipse; #Narrator; #NVDA

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:**
OS: Windows 10 Enterprise (Version 1903, Build: 18362.356)
Screen Reader: Narrator, NVDA
MAS Violated: MAS1.3.1

**Note:**
**NVDA is announcing the selected list item name twice for the control "Kind" combo box**

**Repro Steps:**
1.Turn on Screen Reader
2.Open Java EE - Eclipse application
3.Install 'Azure Toolkit for Eclipse' from Marketplace
4.Open Eclipse and install  the Scala plug-in 
5.Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
6.Tab till File-->New-->Project and press enter. Tab till HDInsight and expand it. Select 'Spark on HDInsight (Scala)'. Tab till Next and press enter
7.Tab till Next button and press enter. Press Finish button
8.In Project Explorer, Expand the created scala project right-click src, New-->Other and press enter
9.Tab till Scala Wizards, expand and select 'Scala Object' and press next
10.Tab till the Combo box "Kind"
11.Verify whether screen reader is announcing the role as Combo box or not

**Actual Result:**
Narrator is announcing as "Scala Object" 0% vertically scrolled, but not reading the role as Combo box

**Expected Result:**
Narrator/NVDA should announce as Scala Object, Kind Combo box Collapsed

**User Impact:**
Screen Reader users will not understand about the type of control focus is going, if it doesn't announce any information about it's role

**MAS Reference**:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug4_Narrator is not announcing the role as combo box when focus lands on scala object control.pptx](https://github.com/microsoft/azure-tools-for-java/files/3635821/Bug4_Narrator.is.not.announcing.the.role.as.combo.box.when.focus.lands.on.scala.object.control.pptx)
![Bug4_NVDA is announcing the selected list item name twice](https://user-images.githubusercontent.com/51435937/65594540-a828e000-dfb0-11e9-9185-acb31dee9f17.png)

**Comment by t-rufang** [comment]
It is determined by Eclipse IDE. We cannot fix it. And it's out of the toolkit's scope.

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749737
